### PR TITLE
add kibana minio postgres prometheus kafka

### PR DESCRIPTION
**Pull request by SnakeHacker**

### CODE REVIEW

**Comment by SnakeHacker** [comment]
@iminders PTAL

File: starter/grafana/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+version: '3'
+
+services:
+    grafana:
+      image: grafana/grafana:6.7.2
```
Comment (iminders) [inline starter/grafana/docker-compose.yml:5]: restart: always

File: starter/postgres/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+version: '3'
+
+services:
+    postgres:
+        image: postgres:11.1
+        environment:
```
Comment (iminders) [inline starter/postgres/docker-compose.yml:6]: restart: always

**Review by iminders — CHANGES_REQUESTED** [review]

File: starter/postgres/docker-compose.yml (hunk 2 above)
Comment (SnakeHacker) [reply starter/postgres/docker-compose.yml:6]: Done

File: starter/grafana/docker-compose.yml (hunk 1 above)
Comment (SnakeHacker) [reply starter/grafana/docker-compose.yml:5]: Done

**Comment by iminders** [comment]
LGTM

**Review by iminders — APPROVED** [review]

**Pull request merged**